packages/web-core/src/services/ConnectService.ts

```diff
@@ -375,4 +382,18 @@ export class ConnectService {
   #getDefaultFrontendApiUrl() {
     return `https://${this.#projectId}.${this.#frontendApiUrlSuffix}`;
   }
+
+  getLastLogin() {
+    const lastLogin = ConnectLastLogin.loadFromStorage(this.#projectId);
```

Comment:
isn't this redundant? (loadFromStorage is already returning undefined, right? Or am I missing something?)